### DISCUSSION TITLE
Many (if not all) prebuilt firmware of V1.22.0-preview.236  only 32 KB

### DISCUSSION THREAD

**Discussion by robhamerling** [micropython/micropython#13163] [Core Development]
I noticed that several pre-built firmware files of 1.22.0-preview.236 are only **32 KB** in size. Probably a build problem.

**Comment by scruss** [comment]
… and 32 K of zeroes, at that.

Affected files that I've found:

* ESP8266: [v1.22.0-preview.236.ge1a7aa23f (2023-12-08) .bin](https://micropython.org/resources/firmware/ESP8266_GENERIC-20231208-v1.22.0-preview.236.ge1a7aa23f.bin)
* RPI Pico: [v1.22.0-preview.236.ge1a7aa23f (2023-12-08) .uf2](https://micropython.org/resources/firmware/RPI_PICO-20231208-v1.22.0-preview.236.ge1a7aa23f.uf2)
* pyboard 1.1: [v1.22.0-preview.236.ge1a7aa23f (2023-12-08) .dfu](https://micropython.org/resources/firmware/PYBV11-20231208-v1.22.0-preview.236.ge1a7aa23f.dfu) - the hex file is the same

**Comment by robhamerling** [comment]
Problem seems be solved with firmware files dd december 10, 2023

**Comment by mattytrentini** [comment]
Yes, it seems the build server's hard drive was full - thanks for looking into this on a Sunday morning @dpgeorge!

(The problematic build artefacts have been deleted so others don't inadvertently used them - which is why the links above are now broken.)
